src/components/Nav.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+import Logo from '../icon.png';
+
+const Nav = () => (
+  <header className="header">
+    <span className="head">
+      <img className="img navbar-brand" src={Logo} alt="logo" />
+      <h1>
+        {'Space Travelers\'s Hub'}
+      </h1>
+    </span>
+    <nav className="nav justify-content-end">
+      <NavLink to="/" className="nav-link" activeClassName="is-active" exact>
+        <span className="rockets">Rockets</span>
+      </NavLink>
+      <NavLink
+        to="/missions"
+        className="nav-link"
+        activeClassName="is-active"
```

Comment:
[OPTIONAL] You have a `is-active` class, but it's not styled. I think you could put a text-decoration and a darker color, so the user knows in which section of the project is.